PixiEditor/Models/IO/Exporter.cs

```diff
@@ -48,22 +52,35 @@ public static string SaveAsEditableFile(Document document, string path)
             return path;
         }
 
+        //static Dictionary<ImageFormat, Action<ExportFileDialog, WriteableBitmap>> encoders = new Dictionary<ImageFormat, Action<ExportFileDialog, WriteableBitmap>>();
+        //TODO remove static methods/members
+        static Dictionary<ImageFormat, Func<BitmapEncoder>> encodersFactory = new Dictionary<ImageFormat, Func<BitmapEncoder>>();

```

Comment:
This is weird, can't you just create Dictionary of <ImageFormat, BitmapEncoder> and instantiate them if null?

Reply:
Both approach would do the job. Current one creates a transient BitmapEncoder, in your proposal it will stay in memory. 
Since memory footprint is small here I would change to your approach as it's a bit simpler code there.